plugin/pkg/healthcheck/healthcheck.go

```diff
@@ -212,6 +212,10 @@ func (u *HealthCheck) Select() *UpstreamHost {
 
 // normalizeCheckURL creates a proper URL for the health check.
 func (u *HealthCheck) normalizeCheckURL(name string) string {
+	if u.Path == "" {
```

Comment:
That is a good test to avoid to check an unwanted path.

Reply:
I see the point : HealthCheckURL is called directly by proxy, even though the workers are not triggered in Start() because path is empty.